client/client.go

```diff
@@ -145,13 +146,23 @@ type SecurityOption struct {
 	KeyPath  string
 }
 
+// ClientOption configures client.
+type ClientOption func(c *client)
+
+// WithGRPCDialOptions configures the client with gRPC dial options.
```

Comment:
For now, we have no `gRPC` related parameter exposed in `Client`, to keep the interface implementation-independent, how about defining a new `Option` type and map it to `grpc.DialOption`?

Reply:
That's means if I want to add a new gRPC option, I need to map it again and then update pd in tidb again.

Reply:
you can check etcd client also exposed gRPC option:

https://github.com/etcd-io/etcd/blob/5770a6d286fe48682e29b54ce0df37e7d24b3280/clientv3/config.go#L70-L73